backend/open_webui/routers/files.py

```
@@ -84,19 +86,32 @@ def has_access_to_file(
 def upload_file(
```

Ruff (0.11.9)

`upload_file` is too complex (12 > 10)

(C901)